apps/fz_http/lib/fz_http_web/live/rule_live/rule_list_component.html.heex

```diff
@@ -40,6 +49,9 @@
                   <%= rule.destination %>
                 </dd>
               </td>
+              <td class="has-text-left">
+                <%= @users[rule.user_id] %>
```

Comment:
Will there be rule.user_id == current_user_id?

Reply:
The map filters out the current user id before being built.:

https://github.com/firezone/firezone/blob/21394ca539b147b088d41914ff742e846b3fed26/apps/fz_http/lib/fz_http_web/live/rule_live/rule_list_component.ex#L74-L77

Reply:
I thought we'd want the current user in the map. Maybe I didn't get what this is trying to achieve.

Reply:
Yeah, hmmm 🤔 

The current user would be the admin, so I was assuming as an admin you would only scope rules to your users and not yourself?

Reply:
What if the admin wants some special sauce for themself :)

Also an admin could add a rule for another admin, right?
In that case, you wouldn't want to mistake a rule that only applies to you as global.

Reply:
Right, that makes more sense. Thanks for clarifying for me!